pytket/setup.py

```diff
@@ -190,21 +190,22 @@ def finalize_options(self):
     license="Apache 2",
     packages=setuptools.find_packages() + ["pytket.qasm.includes"],
     install_requires=[
-        "sympy ~=1.6",
-        "numpy >=1.21.4, <2.0",
-        "lark ~=1.1",
-        "scipy ~=1.13",
+        "sympy >= 1.12.1",
```

Comment:
Are we sure that we want to do this now and not wait a bit how this works on pytket-qiskit?

Reply:
I actually think it's worth doing now. Updating the lower bounds is something we've always done as a matter of course (when we remember), and removing the upper bounds (and fixing issues as they arise) is better than forcing users to have a lower version than they need (and than may be required by other packages).